src/modules/Y2UsersLinux.rb

```diff
@@ -30,20 +30,27 @@ module Yast
   class Y2UsersLinuxClass < Module
     include Yast::Logger
 
-    EXPORTED_USERADD_ATTRS =
-      ["group", "home", "inactivity_period", "expiration", "shell", "skel", "umask"].freeze
+    EXPORTED_USERADD_ATTRS = [
+      "group", "home", "shell", "skel", "umask", { "inactivity_period" => "inactive" },
+      { "expiration" => "expire" }
```

Comment:
this is kind of tricky structure. I think it should be either documented or changed to hash that is mapping ( even if there is same key and value )